content/sensu-enterprise/2.6/guides/using-contact-routing.md

```diff
@@ -0,0 +1,623 @@
+---
+title: "Contact Routing Guide"
+linkTitle: "Contact Routing Guide"
+description: "Set up a Sensu Enterprise Contact Routing."
+product: "Sensu Enterprise"
+version: "2.6"
+weight: 7
+menu:
+ sensu-enterprise-2.6:
+   parent: guides
+---
+
+- [Prerequisites](#prerequisites)
+- [Overview](#overview)
+  - [Contact Routing Basics](#contact-routing-basics)
+    - [Contact Configuration](#contact-configuration)
+    - [Check Configuration](#check-configuration)
+    - [Client Configuration](#client-configuration)
+- [Example Implementation](#example-implementation)
+  - [Default Configuration](#default-configuration)
+  - [Single Handler](#single-handler)
+      - [Single Handler with a Single Matching Contact](#single-handler-with-a-single-matching-contact)
+      - [Single Handler with a Single Non-Matching Contact](#single-handler-a-single-non-matching-contact)
+      - [Single Handler with Multiple Matching Contacts](#single-handler-with-multiple-matching-contacts)
+      - [Single Handler with Multiple Non-Matching Contacts](#single-handler-with-multiple-non-matching-contacts)
+      - [Single Handler with Some Matching Contacts](#single-handler-with-some-matching-contacts)
+  - [Multiple Handlers](#multiple-handlers)
+      - [Multiple Handlers with a Matching Contact](#multiple-handlers-with-a-matching-contact)
+      - [Multiple Handlers with a Single Matching Contact](#multiple-handlers-with-a-single-matching-contact)
+      - [Multiple Handlers with a Single Non-Matching Contact](#multiple-handlers-with-a-single-non-matching-contact)
+      - [Multiple Handlers with Multiple Matching Contacts](#multiple-handlers-with-multiple-matching-contacts)
+      - [Multiple Handlers with Multiple Non-Matching Contacts](#multiple-handlers-with-multiple-non-matching-contacts)
+      - [Multiple Handlers With Some Matching Contacts](#multiple-handlers-some-matching-contacts)
+- [Wrapping Up](#wrapping-up)
+  - [Contact Routing and Sensu Event Pipeline](#contact-routing-and-sensu-event-pipeline)
+- [References](#references)
+- [Additional Resources](#additional-resources)
+
+# Prerequisites
+
+Before diving into this guide, we recommend having the following components ready:
+
+- A working Sensu Enterprise deployment
+- Two or more handlers configured
+
+If you've not already signed up for Sensu Enterprise, you can do so via [this link][1].
+
+In this guide we'll be using [Slack][2] and [Email][3] handlers for our demonstration.
+
+# Overview
+
+Every incident, outage or event has an ideal first responder. This can be either
+a team or individual with the knowledge to tirage and address the issue. Sensu
+Enterprise contact routing makes it possible to assign checks to specific teams
+and/or individuals, reducing mean time to response and recovery (MTTR). Contact
+routing works with all of the Sensu Enterprise third-party notification and
```

Comment:
Does conctact routing work for non-Enterprise integrations? If so, worth a mention as that continues to add value for community based handlers or ones that a customer may have written themselves.